src/bn_wallets.erl

```diff
@@ -199,7 +199,7 @@ handle_rpc(<<"wallet_pay_multi">>, {Param}) ->
         case ?jsonrpc_get_param(<<"payments">>, Param, false) of
             L when is_list(L) andalso length(L) > 0 ->
                 lists:map(
-                    fun(Entry) ->
+                    fun({Entry}) ->
```

Comment:
This list of payments is really being constructed as a list of single-element tuples?

Reply:
Believe it or not, it really is. This is broken, pre token-additions, even on master right now as a result. See PR comment about it, above.